r1cs/gadgets/crypto/src/commitment/pedersen/mod.rs

```diff
@@ -201,12 +203,14 @@ mod test {
     };
     use r1cs_core::ConstraintSystem;
     use r1cs_std::{
-        instantiated::jubjub::JubJubGadget, prelude::*,
+        instantiated::tweedle::TweedleDeeGadget, prelude::*,
         test_constraint_system::TestConstraintSystem,
     };
     use rand::thread_rng;
 
+    // TODO: Test should be updated and fixed for tweedle
     #[test]
+    #[ignore]
```

Comment:
The error looks like this:

---- commitment::pedersen::test::commitment_gadget_test stdout ----
thread 'commitment::pedersen::test::commitment_gadget_test' panicked at 'assertion failed: `(left == right)`
  left: `Fp256(BigInteger256([1527091739389175160, 9187061243545886536, 6292075612643128339, 1411436306187210958]))`,
 right: `Fp256(BigInteger256([4622220868924625837, 1391346539492113811, 13332386268608611785, 1349341222340786515]))`', r1cs/gadgets/crypto/src/commitment/pedersen/mod.rs:262:9

Reply:
Test fails due to incomplete arithmetic.
I've updated issue #125 about this.
We should rewrite the precomputed_base_multi_scalar_mul to work with SW curve gadgets (currently the only ones we have), either with incomplete or complete arithmetic (we can optimize it too as we have our far more efficient mul_bits_fixed_base that we can use instead of the approach of the default implementation). I've taken a couple of actions:

1. Removed Pedersen commitment/hash that we never use most likely and replaced its occurences with BoweHopwood hash instead (that works even with incomplete arithmetic thanks to ZCash algorithm). Before, BH was not behaving exactly as a FixedLength hash as the input was padded only to the next multiple of the chunk size. Now, it is always padded to have length WINDOW_SIZE * NUM_WINDOWS * CHUNK_SIZE. We should check all its usages (Base MerkleTree and ECVRF) to assess that the current construction is secure (I'm refering to collision resistance here mostly) @UlrichHaboeck75 ;
2. Removed default implementation of precomputed_base_multi_scalar_mul (that doesn't work now) and precomputed_base_scalar_mul (it's a duplicate of our mul_bits_fixed_base which is far more efficient).